packages/fluentui/docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -38,9 +40,9 @@ type ChatItem = {
 type StatusPropsExtendable = Extendable<StatusProps>;
 
 const statusMap: Map<UserStatus, StatusPropsExtendable> = new Map([
-  ['Available', { color: 'green', icon: 'check', title: 'Available' }],
-  ['DoNotDisturb', { color: 'red', icon: 'minus', title: 'Do not disturb' }],
-  ['Away', { color: 'yellow', icon: 'clock', title: 'Away' }],
+  ['Available', { color: 'green', icon: <Icon name="check" />, title: 'Available' }],
```

Comment:
```suggestion
  ['Available', { color: 'green', icon: <AcceptIcon />, title: 'Available' }],
```
Please shouldn't this be AcceptIcon?